saleor/warehouse/management.py

```diff
@@ -126,7 +126,6 @@ def _create_allocations(
         return insufficient_stock, []
 
 
-@traced_atomic_transaction()
```

Comment:
Do we remove it because it's always called in a transaction?

Reply:
Otherwise, the `Allocation` changes are not saved. And yes, this method is only called in transactions.